src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -490,3 +492,11 @@ def excel2properties(
             print(f"properties section was created successfully and written to file '{path_to_output_file}'")
 
     return props, True
+
+
+def _read_check_property_df(excelfile: str) -> pd.DataFrame | None:
```

Comment:
I would rename this to `read_and_check_property_df`

Reply:
why? this is only used in this file and nowhere else...

Reply:
Sorry, misunderstanding: I didn't mean to omit the _, I was rather pointing at the wording. So my suggestion is to rename it to `_read_and_check_property_df`.